discord_bots/cogs/economy.py

```diff
@@ -1015,6 +1023,12 @@ async def on_submit(self, interaction: Interaction[Client]) -> None:
             )
             return
 
+        if self.value < 0:
```

Comment:
Should be `<=` so you can't bet exactly 0, right?

Reply:
[6a7204c](https://github.com/tribesonecommunity/discord-bots/pull/64/commits/6a7204c2d16e498dd7eb1b6b344e94f7b47f76e5)
Changed to `<=`